### ISSUE TITLE
Samsung Galaxy Tab 4 7.0 SM-T231

### ISSUE THREAD

**Issue by susl16c** [mgks/Android-SmartWebView#24]
Samsung Galaxy Tab 4 7.0 SM-T231 - Prompt when the application starts on this device appears only white screen, how can I fix it?

> 05-10 09:07:19.752 2614-2614/mgks.os.webview E/EGL_emulation: tid 2614: eglSurfaceAttrib(1199): error 0x3009 (EGL_BAD_MATCH)
05-10 09:07:24.592 2614-2614/mgks.os.webview E/dalvikvm: Could not find class 'android.app.ActivityManager$TaskDescription', referenced from method mgks.os.webview.MainActivity.onResume
05-10 09:07:24.632 2614-2614/mgks.os.webview E/dalvikvm: Could not find class 'android.graphics.drawable.RippleDrawable', referenced from method android.support.v7.widget.AppCompatImageHelper.hasOverlappingRendering
05-10 09:07:24.762 2614-2614/mgks.os.webview E/EGL_emulation: tid 2614: eglSurfaceAttrib(1199): error 0x3009 (EGL_BAD_MATCH)

**Comment by susl16c** [comment]
Yes it is absolutely true, but on it it is impossible to eliminate such a problem?

**Comment by mgks** [comment]
Can update us on the issue? Have you figured out any way to solve it?

**Comment by mgks** [comment]
Hi,
can you update us regarding the issue?
